horizons/ai/aiplayer/mission/specialdomestictrade.py

```diff
@@ -114,8 +118,12 @@ def _reached_source_settlement(self):
 			self.report_failure('No resources to transport')
 
 	def _move_to_destination_settlement(self):
-		self._move_to_warehouse_area(self.destination_settlement_manager.settlement.warehouse.position, Callback(self._reached_destination_settlement),
```

Comment:
I think the change itself is fine, I'd just like to see this split up into two commits next time.